cinder/README.md

```diff
@@ -183,7 +186,7 @@ For a full detailed instruction of options, please refer to [Unity backend confi
 
 ```bash
 (undercloud) $ openstack overcloud deploy --templates \
-  -e /home/stack/templates/overcloud_images.yaml \
+  -e /home/stack/templates/containers-prepare-parameter.yaml \
   -e /home/stack/templates/custom-dellemc-container.yaml \
   -e /home/stack/templates/cinder-backend-dellemc-unity.yaml \
   -e <other templates>
```

Comment:
A general question: does OSP16 support configuring certificates for Cinder backend?

Reply:
I remember no such configuration for Cinder Unity/VNX, right?

Reply:
These are configurations for ssl verification and certificate. You could check the codes.

Reply:
You mean driver_ssl_cert_verify and driver_ssl_cert_path defined in Cinder framework? We didn't add these options, we only added the options defined in our drivers.

Reply:
Yes. The user can set these two options when he/she wants to enable ssl verify. All the options defined in Cinder framework can be used in the backend. So, I am wondering how does OSP specify the driver_ssl_cert_path in the container. It is a file path. How is this file on the host mounted into the container? Does RH have offical way to do this? Do we have any best practice for this?

Reply:
We only cover very few scenarios in our verification, to answer these questions, we need to spend more time to investigate them, this is another user story, we couldn't resolve them in this document.

Reply:
OK. Please add it to the backlog.